docs/components/modeler/desktop-modeler/telemetry/telemetry.md

```diff
@@ -193,3 +193,22 @@ Example event:
   "triggerType": "keyboard"
 }
 ```
+
+## Task testing events
+
+`Task Testing` events are sent when using [task testing](../task-testing.md) feature:
```

Comment:
```suggestion
Task testing events are sent when using [task testing](../task-testing.md) feature:
```

Reply:
To be honest, I don't know why we started using `code` for the events on this page. The event names are not code.